apps/code-examples/src/app/shared/theme-selector/theme-selector.component.html

```diff
@@ -56,7 +56,6 @@
     [disabled]="brandingValues().length === 0"
     [(ngModel)]="themeBrand"
   >
-    <option [ngValue]="undefined">None</option>
```

Comment:
I think default theme defaults to the `None` option but it's probably fine if it is just blank now